Shopify/shopify-cli-extensions
# Component: ActionSet, Action, ActionSpacer

issue: #123 

## Testing
1. Run `yarn storybook`
2. Go to `http://localhost:6006/`
3. You should see this:
https://user-images.githubusercontent.com/90475219/139957407-85c55ed8-eafc-4474-950c-7703829c9ad2.mov


## Note
The Actions currently does not show the action when you click on each action. This will be further discussed.
